diff -up ghdl/src/ortho/llvm4-nodebug/Makefile.llvmflags~ ghdl/src/ortho/llvm4-nodebug/Makefile
--- ghdl/src/ortho/llvm4-nodebug/Makefile.llvmflags~	2020-01-29 09:40:38.264921221 +0100
+++ ghdl/src/ortho/llvm4-nodebug/Makefile	2020-01-29 09:42:36.380073411 +0100
@@ -17,7 +17,7 @@ $(ortho_exec): $(ortho_srcdir)/llvm4-nod
 	`$(LLVM_LDFLAGS)` $(LDFLAGS)
 
 llvm-cbindings.o: $(ortho_srcdir)/llvm4-nodebug/llvm-cbindings.cpp
-	$(CXX) -c `$(LLVM_CONFIG) --cxxflags` -o $@ $<
+	$(CXX) -c `$(LLVM_CONFIG) --cxxflags|sed -e s,-fstack-clash-protection,,` -o $@ $<
 
 clean:
 	$(RM) -f *.o *.ali ortho_code_main
